lib/Widgets/DatePicker.vala

```diff
@@ -51,8 +51,18 @@ namespace Granite.Widgets {
 
         private GLib.DateTime _date;
 
+        private GLib.DateTime today;
+
         private bool proc_next_day_selected = true;
 
+        private const string CSS = """
+            calendar
+            {
+                background: transparent;
+                padding: 3px;
+            }
+        """;
+
```

Comment:
Styles are accounted for in the system stylesheet already, so we shouldn't have them here